### PR TITLE
Read msgraph user.select from configuration

### PR DESCRIPTION
**Pull request by LauraMoraB**
## Hey, I just made a Pull Request!

**Description**
Read `user.select` field from the configuration (app-config.yml) as it is currently not loaded.

**Context**
The current documentation suggests that users' msgraph fields can be selected using the `user.select` field: 
image

However, `user.select` was never loaded in `readProviderConfig()` - though it was expected in [read.ts](https://github.com/backstage/backstage/blob/master/plugins/catalog-backend-module-msgraph/src/microsoftGraph/read.ts#L51).

#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [x] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: .changeset/silly-buckets-joke.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-catalog-backend-module-msgraph': patch
+---
+
+Fixed bug in readProviderConfig as it wasn't reading the user.select field expected from the app-config.yaml configuration
```
Comment (awanlin) [inline .changeset/silly-buckets-joke.md:5]: I have a feeling that Vale the "spell checker" might not like some of these. The easy fix is to just use backticks (`) around: readProviderConfig, user.select and app-config.yaml. There's a GitHub action for this that will let you know for sure which words

**Review by awanlin — COMMENTED** [review]
Thanks for the contribution @LauraMoraB 🚀 

Just left a small comment about the changeset everything else looks good to me :)

File: .changeset/silly-buckets-joke.md (hunk 1 above)
Comment (awanlin) [reply .changeset/silly-buckets-joke.md:5]: Looks like I'm happily wrong :)

**Review by awanlin — APPROVED** [review]

**Review by benjdlambert — APPROVED** [review]
Hey :wave: thanks for this! Looks great - I just changed the changeset a little bit, but good to go when it's green! :pray:

**Pull request merged**